datafusion/common/src/config.rs

```diff
@@ -325,11 +325,7 @@ config_namespace! {
         /// Aggregation ratio (number of distinct groups / number of input rows)
         /// threshold for skipping partial aggregation. If the value is greater
         /// then partial aggregation will skip aggregation for further input
-        pub skip_partial_aggregation_probe_ratio_threshold: f64, default = 0.8
-
-        /// Number of input rows partial aggregation partition should process, before
-        /// aggregation ratio check and trying to switch to skipping aggregation mode
-        pub skip_partial_aggregation_probe_rows_threshold: usize, default = 100_000
```

Comment:
Why has this setting been deleted? Checking if aggregation should be skipped right in the beginning of the execution may lead to skipping decision made based on insufficient amount of data.

Reply:
The reason is that in my strategy I make the decision per batch since I assume the data is distributed evenly

Reply:
config with 0.1 and 100_000
```
┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Query        ┃       main ┃  threshold ┃        Change ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ QQuery 0     │     0.41ms │     0.45ms │  1.10x slower │
│ QQuery 1     │    42.61ms │    42.55ms │     no change │
│ QQuery 2     │    75.89ms │    74.88ms │     no change │
│ QQuery 3     │    72.24ms │    61.81ms │ +1.17x faster │
│ QQuery 4     │   399.04ms │   359.82ms │ +1.11x faster │
│ QQuery 5     │   663.20ms │   699.03ms │  1.05x slower │
│ QQuery 6     │    37.91ms │    38.01ms │     no change │
│ QQuery 7     │    42.71ms │    40.05ms │ +1.07x faster │
│ QQuery 8     │   638.09ms │   699.07ms │  1.10x slower │
│ QQuery 9     │   678.82ms │   742.56ms │  1.09x slower │
│ QQuery 10    │   200.65ms │   209.83ms │     no change │
│ QQuery 11    │   233.85ms │   220.56ms │ +1.06x faster │
│ QQuery 12    │   718.91ms │   660.93ms │ +1.09x faster │
│ QQuery 13    │   926.86ms │   937.19ms │     no change │
│ QQuery 14    │   873.60ms │   753.51ms │ +1.16x faster │
│ QQuery 15    │   498.70ms │   483.92ms │     no change │
│ QQuery 16    │  1305.01ms │  1238.38ms │ +1.05x faster │
│ QQuery 17    │  1190.26ms │  1111.67ms │ +1.07x faster │
│ QQuery 18    │  3347.18ms │  2903.17ms │ +1.15x faster │
│ QQuery 19    │    55.77ms │    61.39ms │  1.10x slower │
│ QQuery 20    │   943.13ms │   919.37ms │     no change │
│ QQuery 21    │  1206.45ms │  1230.95ms │     no change │
│ QQuery 22    │  3230.71ms │  3479.20ms │  1.08x slower │
│ QQuery 23    │  8186.73ms │  7989.35ms │     no change │
│ QQuery 24    │   500.91ms │   507.13ms │     no change │
│ QQuery 25    │   502.13ms │   551.57ms │  1.10x slower │
│ QQuery 26    │   561.53ms │   568.74ms │     no change │
│ QQuery 27    │  1338.06ms │  1382.16ms │     no change │
│ QQuery 28    │ 10507.08ms │ 11253.66ms │  1.07x slower │
│ QQuery 29    │   398.16ms │   429.90ms │  1.08x slower │
│ QQuery 30    │   760.72ms │   664.19ms │ +1.15x faster │
│ QQuery 31    │   712.91ms │   772.65ms │  1.08x slower │
│ QQuery 32    │  3567.36ms │  4167.37ms │  1.17x slower │
│ QQuery 33    │  4863.25ms │  4311.27ms │ +1.13x faster │
│ QQuery 34    │  4115.97ms │  3564.32ms │ +1.15x faster │
│ QQuery 35    │   998.40ms │   979.18ms │     no change │
│ QQuery 36    │   147.99ms │   136.97ms │ +1.08x faster │
│ QQuery 37    │   101.82ms │   102.79ms │     no change │
│ QQuery 38    │   108.77ms │   107.66ms │     no change │
│ QQuery 39    │   327.29ms │   272.91ms │ +1.20x faster │
│ QQuery 40    │    33.56ms │    34.88ms │     no change │
│ QQuery 41    │    34.09ms │    32.18ms │ +1.06x faster │
│ QQuery 42    │    39.32ms │    39.75ms │     no change │
└──────────────┴────────────┴────────────┴───────────────┘
```

config with 0.1 and 0
```
┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Query        ┃       main ┃  threshold ┃        Change ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ QQuery 0     │     0.41ms │     0.49ms │  1.21x slower │
│ QQuery 1     │    42.61ms │    42.34ms │     no change │
│ QQuery 2     │    75.89ms │    81.91ms │  1.08x slower │
│ QQuery 3     │    72.24ms │    69.94ms │     no change │
│ QQuery 4     │   399.04ms │   365.30ms │ +1.09x faster │
│ QQuery 5     │   663.20ms │   656.21ms │     no change │
│ QQuery 6     │    37.91ms │    38.24ms │     no change │
│ QQuery 7     │    42.71ms │    40.27ms │ +1.06x faster │
│ QQuery 8     │   638.09ms │   590.64ms │ +1.08x faster │
│ QQuery 9     │   678.82ms │   666.57ms │     no change │
│ QQuery 10    │   200.65ms │   196.94ms │     no change │
│ QQuery 11    │   233.85ms │   220.90ms │ +1.06x faster │
│ QQuery 12    │   718.91ms │   652.58ms │ +1.10x faster │
│ QQuery 13    │   926.86ms │   941.22ms │     no change │
│ QQuery 14    │   873.60ms │   749.50ms │ +1.17x faster │
│ QQuery 15    │   498.70ms │   495.09ms │     no change │
│ QQuery 16    │  1305.01ms │  1247.99ms │     no change │
│ QQuery 17    │  1190.26ms │  1158.71ms │     no change │
│ QQuery 18    │  3347.18ms │  2008.84ms │ +1.67x faster │
│ QQuery 19    │    55.77ms │    54.94ms │     no change │
│ QQuery 20    │   943.13ms │   913.04ms │     no change │
│ QQuery 21    │  1206.45ms │  1183.49ms │     no change │
│ QQuery 22    │  3230.71ms │  3181.47ms │     no change │
│ QQuery 23    │  8186.73ms │  7900.35ms │     no change │
│ QQuery 24    │   500.91ms │   501.30ms │     no change │
│ QQuery 25    │   502.13ms │   481.08ms │     no change │
│ QQuery 26    │   561.53ms │   571.92ms │     no change │
│ QQuery 27    │  1338.06ms │  1451.63ms │  1.08x slower │
│ QQuery 28    │ 10507.08ms │ 11396.72ms │  1.08x slower │
│ QQuery 29    │   398.16ms │   392.46ms │     no change │
│ QQuery 30    │   760.72ms │   658.78ms │ +1.15x faster │
│ QQuery 31    │   712.91ms │   702.45ms │     no change │
│ QQuery 32    │  3567.36ms │  3520.95ms │     no change │
│ QQuery 33    │  4863.25ms │  3527.36ms │ +1.38x faster │
│ QQuery 34    │  4115.97ms │  3731.86ms │ +1.10x faster │
│ QQuery 35    │   998.40ms │   964.65ms │     no change │
│ QQuery 36    │   147.99ms │   136.33ms │ +1.09x faster │
│ QQuery 37    │   101.82ms │   102.70ms │     no change │
│ QQuery 38    │   108.77ms │   105.97ms │     no change │
│ QQuery 39    │   327.29ms │   267.66ms │ +1.22x faster │
│ QQuery 40    │    33.56ms │    34.77ms │     no change │
│ QQuery 41    │    34.09ms │    31.76ms │ +1.07x faster │
│ QQuery 42    │    39.32ms │    40.56ms │     no change │
└──────────────┴────────────┴────────────┴───────────────┘
```